fastfood/templating.py

```diff
@@ -57,7 +58,9 @@ def render_templates_generator(*files, **template_map):
                              % os.path.relpath(path))
         else:
             try:
-                template = JINJA_ENV.from_string(open(path).read())
+                with codecs.open(path, encoding='utf-8') as f:
```

Comment:
Yeah.... 

http://nedbatchelder.com/text/unipain/unipain.html#18

Starting at slide 18 he talks about the differences between python 2 and 3 WRT unicode/strings/bytearrays.